content/js/index.md

```diff
@@ -1,65 +1,142 @@
+<br/><br/>
 <a name="install-js-sdk"></a>
-## Install Skygear JS SDK
+# Install Skygear JS SDK
 
-There are several ways to install Skygear SDK to you app. You can either:
+There are several ways to install Skygear JS SDK to you app. You can either:
 
-* Use the latest stable release on npm
-* Install the latest development version
+- Install the stable release on npm
+- Install the latest development version
 
-### Use the latest stable release on npm
-You can install Skygear JS SDK via [npm](https://www.npmjs.com/package/skygear). The SDK  package is available on the npm website.
+## Install the stable release on npm
 
+You can install Skygear JS SDK via [npm](https://www.npmjs.com/package/skygear).
+The SDK package is available on the npm website.
 
 ``` bash
 npm install skygear --save
 ```
 
-### Install the latest development version
+## Install the latest development version
 
 You can add the development branch GitHub Repository as a submodule in your project.
 
 ``` bash
-git submodule git@github.com:skygeario/skygear-server.git
+git submodule add https://github.com/SkygearIO/skygear-SDK-JS.git skygear
 cd skygear
 npm install
 cd ..
 npm install skygear/
 ```
 
+
+
+
+
+
+
+
+
+
+<br/><br/>
 <a name="include-js-sdk"></a>
-## Include Skygear JS SDK in your project
+# Include Skygear JS SDK in your project
 
-### Include in your npm project
+## Include in your npm project
 
-The skygear JS SDK is assumed to be run in ES6 runtime. If you wish to run it in an older runtime, you need to require the [polyfill plugin](https://babeljs.io/docs/usage/polyfill/) before importing the Skygear SDK.
+The Skygear JS SDK is assumed to be run in ES6 runtime. If you wish to run it in
+an older runtime, you need to require the [polyfill plugin](https://babeljs.io/docs/usage/polyfill/)
+before importing the Skygear SDK.
 
-``` js
+``` javascript
 import 'babel-polyfill';
 import skygear from 'skygear';
 ```
 
-### Include in React Native projects
-You can also include Skygear JS SDK in your React Native project.
+## Include in your website
 
-``` js
-require('babel-polyfill');
-var React = require('react-native');
+Build the Skygear module first (make sure all the tests pass,
+if not please check your node version or report an issue):
 
-/* ... */
-
-var skygear = require('skygear');
+``` bash
+cd skygear
+npm test
+cd ..
 ```
```

Comment:
You will got a direct link after this issue is resolved. Let remove this npm test first.